basics/a11y.md

```diff
@@ -2,67 +2,94 @@
-  issues before or shortly after a project goes into production.
-
-## Implementing Open Source or Vended Solutions
-
-- Discuss with UX whether an accessibility review is appropriate and who will
-  lead the review.
-
-## Automated Checking
-
-- All of our public GitHub repos are set to automatically check via
-  [AccessLint](https://www.accesslint.com), which is great for what it is
-  but is _not_ a comprehensive Accessibility Plan.
-
-## Manual testing
-
-- The [Web Accessibility Evaluation Tool (WAVE)](https://wave.webaim.org/) is
-  useful for checking single web pages. It can be installed as a browser
-  extension for either Chrome or Firefox.
-- The [Accessible Name & Description Inspector (ANDI)
-  tool](https://www.ssa.gov/accessibility/andi/help/install.html) is a
-  Javascript tool to evaluate a loaded webpage.
-- A screenreader like [VoiceOver](https://www.apple.com/voiceover/info/guide/_1121.html)
+### Accessibility testing
+
+#### Automated testing
+
+All of our public GitHub repos are set to automatically check via [AccessLint](https://www.accesslint.com). This is a
+useful tool, but is _not_ a comprehensive accessibility plan.
+
+#### Manual testing
+
+- For every pull request that touches a UI element, the author should run an a11y checker tool to confirm that the
+change does not introduce an access barrier. In EngX, we use the following:
+  - The [Web Accessibility Evaluation Tool (WAVE)](https://wave.webaim.org/) is
+    useful for checking single web pages. It can be installed as a browser
+    extension for either Chrome or Firefox.
+  - The [Accessible Name & Description Inspector (ANDI)
+    tool](https://www.ssa.gov/accessibility/andi/help/install.html) is a
+    Javascript tool to evaluate a loaded webpage.
+- Some people navigate webpages by keyboard. To ensure a11y for these users, it's a good idea to attempt to navigate
+  the site periodically using your keyboard (e.g., when a new feature is added to the UI). This can help confirm that
+  UI elements flow in a logical order.
+- A screen reader like [VoiceOver](https://www.apple.com/voiceover/info/guide/_1121.html)
   for the Mac or [NVDA](https://www.nvaccess.org/download/) for Windows can be
   useful for user testing to confirm that elements are reachable, or that
   interactions will play out as expected. Articles like ["Testing with Screen
   Readers" at WebAIM](https://webaim.org/articles/screenreader_testing/) are
   also useful.
+  - While periodic screen reader testing can be helpful to identify issues, it has limited utility unless done by an
+  expert user. (This will occur as part of the formal DAS review.)
+
+## Implementing open source or vended solutions
+
+As with locally developed software, any open source or vended software that includes a UI should be evaluated for
+accessibility. Any vendor claims related to a11y should be confirmed, to the extent we are able to do so. Ideally, a11y
```

Comment:
Maybe a note that UXWS should be contacted for assistance in doing vended UI evaluation?